modules/navigation/navigation_mesh_generator.cpp

```diff
@@ -92,12 +92,15 @@ void NavigationMeshGenerator::_add_mesh(const Ref<Mesh> &p_mesh, const Transform
 		int face_count = index_count / 3;
 
 		Array a = p_mesh->surface_get_arrays(i);
+		ERR_CONTINUE((a.is_empty() || (a.size() != Mesh::ARRAY_MAX)));
```

Comment:
Some extraneous parentheses:
```suggestion
		ERR_CONTINUE(a.is_empty() || (a.size() != Mesh::ARRAY_MAX));
```

Same for several other added conditions.